client/lib/screen-title/index.js

```diff
@@ -13,16 +14,21 @@ import { setTitle as legacySetTitle } from './actions';
 export function subscribeToStore( store ) {
 	let title = store.getState().page.title;
```

Comment:
@aduth If you set the `const state = store.getState()` on this line, it will not receive any updates while in the `store.subscribe` callback. My thought was that this was because the state wasn't filled or didn't return a reference (yet?).